controllers/root.js

```diff
@@ -113,11 +126,16 @@ var Root = (function() {
         };
         proxy.sendData(protocol, options, req.body, res);
     };
+    var redir_request_validate = function (req, res, user_info) {
+            if (user_info) {
+                res.status(200).send('1');
```

Comment:
If the token validation success, the response has to include the information about the user (nickname, roles...) included in `user_info`. Please, send it in the response body